cmake/MiNiFiOptions.cmake

```
@@ -117,6 +123,9 @@ add_minifi_option(ENABLE_TEST_PROCESSORS "Enables test processors" OFF)
 add_minifi_option(ENABLE_PROMETHEUS "Enables Prometheus support." OFF)
 add_minifi_option(DISABLE_JEMALLOC "Disables jemalloc." OFF)
 
+set_minifi_cache_variable(CUSTOM_MALLOC "" "Overwrite malloc implementation.")
+set_property(CACHE CUSTOM_MALLOC PROPERTY STRINGS "jemalloc" "mimalloc" "rpmalloc")
```

Please include a disabled option ("OFF") to be able to disable this on the GUI. I think it would also be better to use it as the default, instead of the empty string.
```suggestion
set_minifi_cache_variable(CUSTOM_MALLOC OFF "Overwrite malloc implementation.")
set_property(CACHE CUSTOM_MALLOC PROPERTY STRINGS "jemalloc" "mimalloc" "rpmalloc" OFF)
```

---

good idea, I've changed it in https://github.com/apache/nifi-minifi-cpp/pull/1456/commits/d08d3c78709c3a1042429de7ac18591909f6e4d2#diff-39f29894b0102815d5fbc5a5c2d97cd04845e983c208dd90ad413c3c7e59b5b0R126-R127